app/_hub/flash/set-dynamic-upstream-host/index.md

```diff
@@ -0,0 +1,123 @@
+---
+name: Set Dynamic Upstream Host
+publisher: Flash
+version: 1.0.0
+
+desc: Constructs the upstream hostname dynamically based on the incoming request parameters
+description: |
+  This plugin can be used to dynamically construct upstream hostname and port number based on the key identifier passed in the incoming request. If the same upstream api is deployed in different servers or data centers, then this plugin can form the hostname for the upstream api dynamically to route it to particular server or data center without making any change in Kong route or service.
+  This plugin extracts the key identifier from the incoming request and following are the supported parameters -
+  * Header param
+  * Query param
+  * Path param
+  * Body param
+  
+  
+support_url: https://github.com/anup-krai/kong-plugin-set-dynamic-target-host/issues
+source_url: https://github.com/anup-krai/kong-plugin-set-dynamic-target-host
+license_type: Apache-2.0 
+
+type: plugin
+categories:
+  - traffic-control
+
+kong_version_compatibility:
+    community_edition:
+      compatible:
+        - 2.4.x
+        - 2.3.x
+        - 2.2.x
+        - 2.1.x
+        - 2.0.x
+        - 1.5.x
+        - 1.4.x
+        - 1.3.x
+        - 1.2.x
+        - 1.1.x
+        - 1.0.x
+    enterprise_edition:
+      compatible:
+        - 2.4.x
+        - 2.3.x
+        - 2.2.x
+        - 2.1.x
+        - 1.5.x
+        - 1.3-x
+
+params:
+  name: set-target-host
+  service_id: true
+  route_id: false
+  consumer_id: false
+  protocols: ["http", "https"]
+  dbless_compatible: true
+    
+  config:
+    - name: upstream_host
+      required: true
+      default:
+      value_in_examples: "nodenumber.org"
```

Comment:
Can we use `example.com` in any domain examples? https://datatracker.ietf.org/doc/html/rfc2606#section-3

Reply:
Updated the domain to example.org in all the domain examples.